LondonBritishLibrary/orient/BLorient686.xml

```diff
@@ -550,39 +550,215 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                                                      <locus from="243r" to="252v"></locus>
+                                                      </item>
+                                                      <item xml:id="q28" n="27">
+                                                        <num value="27">፳፯</num>
+                                                        <dim unit="leaf">10</dim>
+                                                        <locus from="253r" to="262v"></locus>
+                                                            </item>
+                                                            <item xml:id="q29" n="28">
+                                                              <num value="28">፳፰</num>
+                                                              <dim unit="leaf">10</dim>
+                                                              <locus from="263r" to="272v"></locus>
+                                                          </item>
+                                                          <item xml:id="q30" n="29">
+<num value="29">፳፱</num>
+<dim unit="leaf">8</dim>
+<locus from="273r" to="280v"></locus>
+                                                          </item>
+                                                          <item xml:id="q31" n="30">
+                                                            <num value="30">፴</num>
+                                                            <dim unit="leaf">8</dim>
+                                                            <locus from="281r" to="287v"></locus>
+
+                                                          </item>
+                          </list>
+                         </collation>
+                           <condition key="other">The manuscript is generally in a good condition: a few leaves are covered by stains (therefore some passages of the text are not readable), e.g. <locus target="#25v"></locus>, <locus target="#147r"></locus>, <locus target="#262r"></locus>.</condition>
+
                         </supportDesc>
                         <layoutDesc>
                            <layout columns="3" writtenLines="37 39">
                               <ab type="pricking">Pricking and ruling are visible.</ab>
                               <ab type="pricking">Primary pricks and horizontal pricks are visible.</ab>
                               <ab type="ruling" subtype="pattern">1A-1A-1A1A-1A1A/0-0/0-0/C</ab>
-                              <ab type="ruling"> The upper line is written above the ruling,
-                                 the lower line is written above the ruling.</ab>
+                              <ab type="ruling"> The upper line is written above the ruling, the lower line is written above the ruling. The first and last folios are not ruled. On <locus target="#286r"></locus> and <locus target="#286v"></locus> the ruling lines are not competed.</ab>
```

Comment:
What do you mean: "The ruling lines are not competed." Do you mean "completed" or "competed" in a sense of disregarded?

Reply:
thanks, Carsten, I have corrected: it is completed.